server/controllers/authControllers.js

```diff
@@ -0,0 +1,42 @@
+const bcrypt = require('bcrypt');
+const jwt = require('jsonwebtoken');
+const User = require('../../db/schema/userSchema');
+const config = require('config');
+
+const saltRounds = 8;
+
+function generateToken(user) {
+	return jwt.sign(user, config.get('app.token_secret'), {
+		expiresIn: 604800, // in seconds
```

Comment:
Can we make this time as a constant ? In general app level constants to be separated out.

Reply:
@prasanna1211 PR has become large. Can we note it down and add in future fixes?